### ISSUE TITLE
Error: Bash command not found for iOS - possible solution

### ISSUE THREAD

**Issue by nmartinez23** [facebook/react-native#11331]
### Description

I followed the steps to install and set up the dev environment for [React-Native iOS for macOS](https://facebook.github.io/react-native/docs/getting-started.html#content).

The very last step is 'react-native run-ios' on the command line. This should have launched the new app in the Xcode simulator. Instead, I got a "bash: command not found" error message. 

I made sure all of the dependencies were installed and up to date. They all were and I was still getting the error.

### Solution

After searching around, I found [this Stackoverflow post](http://stackoverflow.com/questions/37189081/react-native-command-not-found?noredirect=1&lq=1) to use as a workaround.

Basically, after you enter the command line step: 'npm install -g react-native-cli', look at the output which is the local path. Once you enter the export PATH command that you will see at the Stackoverflow post, just follow the rest of the steps as usual. The last step 'react-native run-ios' will then launch the Xcode simulator.

### Additional Information

* React Native version: 0.39
* Platform: iOS
* Operating System: MacOS

**Comment by hramos** [comment]
We're cutting down on the number of outstanding issues, in order to allow us to focus. I'm closing this issue because it has been open for over 60 days with no activity. If you think it should still be opened let us know why. PRs are always welcome.
